tests/python/unittest/test_operator.py

```diff
@@ -1514,6 +1527,20 @@ def test_nearest_upsampling():
                     check_nearest_upsampling_with_shape(shapes, scale, root_scale)
 
 
+@with_seed()
+def test_bilinear_upsampling():
+    for root_scale in [2,3]:
```

Comment:
you could use `itertools.product` like here - https://github.com/anirudhacharya/incubator-mxnet/blob/master/tests/python/unittest/test_optimizer.py#L1225

Reply:
ok